### PR TITLE
Fix for Support ARGB for pen #393

### PR DESCRIPTION
**Pull request by griffpatch**
Requires fix to scratch-render before this will work.
See:
6f5acfee7b961d71237f1fd50bb3d5a5139c527e

### CODE REVIEW

File: src/blocks/scratch3_pen.js [hunk 1]
```diff
@@ -258,6 +258,9 @@ Scratch3PenBlocks.prototype.setPenColorToColor = function (args, util) {
     penState.penAttributes.color4f[0] = rgb.r / 255.0;
     penState.penAttributes.color4f[1] = rgb.g / 255.0;
     penState.penAttributes.color4f[2] = rgb.b / 255.0;
+    if (rgb.hasOwnProperty('a')) {  // Will there always be an 'a'?
+        penState.penAttributes.color4f[3] = rgb.a/255.0;
```
Comment (thisandagain) [inline src/blocks/scratch3_pen.js]: Looks like you have a lint / code style issue here:
```js
penState.penAttributes.color4f[3] = rgb.a / 255.0;
```

File: src/util/color.js [hunk 2]
```diff
@@ -40,10 +40,11 @@ Color.decimalToHex = function (decimal) {
  * @return {RGBObject} rgb - {r: red [0,255], g: green [0,255], b: blue [0,255]}.
  */
 Color.decimalToRgb = function (decimal) {
+    var a = (decimal >> 24) & 0xFF;
     var r = (decimal >> 16) & 0xFF;
     var g = (decimal >> 8) & 0xFF;
     var b = decimal & 0xFF;
-    return {r: r, g: g, b: b};
+    return {r: r, g: g, b: b, a: a>0 ? a : 255};
```
Comment (thisandagain) [inline src/util/color.js]: and here:
```js
return {r: r, g: g, b: b, a: a > 0 ? a: 255};
```

**Review by thisandagain — CHANGES_REQUESTED** [review]
Looks great! Two very small lint issues that are failing CI. You should be able to test this locally by running `npm test`.

**Comment by griffpatch** [comment]
Ok, I've fixed the lint issues - I also had to add to the test scripts to handle the new alpha values. Do I need to do anything to move the pull request forward?

**Review by thisandagain — APPROVED** [review]

**Comment by thisandagain** [comment]
Thanks @griffpatch !

**Pull request merged**